src/kava.js

```diff
@@ -380,7 +380,8 @@ class Kava extends BasePlugin {
         tabMode: this._isDocumentHidden() ? TabMode.TAB_NOT_FOCUSED : TabMode.TAB_FOCUSED,
         forwardBufferHealth: this._getForwardBufferHealth(),
         targetBuffer: this._getTargetBuffer(),
-        droppedFramesRatio: this._getDroppedFramesRatio()
+        droppedFramesRatio: this._getDroppedFramesRatio(),
+        networkConnectionType: navigator.connection.effectiveType
```

Comment:
not all browsers support `navigator.connection` https://caniuse.com/#feat=netinfo
add protection and need to check what is the default to send in this case.
Also, need to check what are the normative values Kava is accepting, the effectiveType is defined here: https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType

Reply:
in `need to check what is the default to send in this case` I mean, if data is not available, should we even send this attribute or send it conditionally like we do for other conditional information.

Reply:
As far as i talked to Guy before he rather we don't send the property at all if we don't have the data for it

Reply:
Regarding "what are the normative values Kava is accepting" - it is a String... Kava will support whatever we send it. I will add a note in the doc just in case